ROX-9308: Add documentation to remove the admin user (#77226)
openshift/openshift-docs: operating/manage-user-access/remove-admin-user.adoc

:_mod-docs-content-type: ASSEMBLY
[id="remove-admin-user"]
= Remove the admin user
include::modules/common-attributes.adoc[]
:context: remove-admin-user

toc::[]

[role="_abstract"]

{rh-rhacs-first} creates an administrator account, `admin`, during the installation process which can be used to login via username and password.
The password is dynamically generated (unless specifically overridden) and unique to your ACS instance.

In production environments, it is highly recommended to create an authentication provider and remove the `admin` user.

include::modules/remove-admin-user.adoc[leveloffset=+1]

> After applying the configuration changes, once the {product-title-short} portal is available again, login with the `admin` user is not possible anymore.

Reviewer: ```suggestion
After applying the configuration changes, you will not be able to login as an `admin` user.
```

 [NOTE]
 ====
 It is possible to add the `admin` user again as a fallback by simply reverting the configuration changes.

 When enabling the `admin` user again, a new password will be generated.

>  ====

Reviewer: Looking at the procedures, TBH the layout is very awkward. Do you have to have three different procedures? Your first procedure only has one step, as an example. Can this be re-written to have the steps flow together in one procedure, as all of these steps ( I presume) need to be completed to perform the action? 
![image](https://github.com/openshift/openshift-docs/assets/122639474/941b2090-8990-4f71-96ed-40183adcbc66)

Author: It's a bit unfortunate I agree, but they are distinct from one another and only _one_ of them has to be completed.

I can also do something along the lines of:
```
.Procedure
. If you're using the operator:
    . Do step X
. If you're using Helm:
    . Do step Y
```
but didn't want to do this since it felt also a bit awkward - any suggestion what would be the best in case there's steps to be performed based on a choice, but only _one_ choice will be applicable?

Reviewer: Good points. I still think pointing out that the user can choose one method? 
How about something like (remove the sentence that ends with "portal"). 

to remove the `admin` user,  select the applicable installation method. The installation method you select must match the installation method of the RHAS portal.

Also, perhaps I would title each method as "Operator installation method", etc....
